### PR TITLE
web: use Fedora image to build the web interface

### PR DESCRIPTION
**Pull request by TristanCacqueray**
This change should prevent rebuilding ocaml in alpine.

### CODE REVIEW

File: web/Dockerfile [hunk 1]
```diff
@@ -42,16 +42,16 @@ RUN npm run build
 
 ################################################################################
 
-FROM node:12-alpine
+FROM registry.fedoraproject.org/fedora:33
+
+RUN dnf update -y && dnf install -y nodejs && dnf clean all
```
Comment (morucci) [inline web/Dockerfile:47]: We could probably remove this and move the dnf clean all line 20.

**Review by morucci — CHANGES_REQUESTED** [review]
See inline comment please.

File: web/Dockerfile (hunk 1 above)
Comment (TristanCacqueray) [reply web/Dockerfile:47]: How node is going to be installed in the final image? Cleaning builder base image does not seem necessary.

Comment (morucci) [reply web/Dockerfile:47]: Ok I missed about the multi-stage build system. Ok good so let's merge it ! thanks !

**Review by morucci — APPROVED** [review]

**Comment by morucci** [comment]
Tested it on my laptop. It is working well. BTW we could have re-used alpine node for the production stage to avoid the nodejs installation. Anyway let's merge now !

**Pull request merged**